lecture2/task2.c

```diff
@@ -0,0 +1,104 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+typedef struct my_ll {
+	int number;
+	char name[16];
+	struct my_ll *next;
+} myll;
+
+int copy_digits(char *inputs,char *outputs)
+{
+	uint8_t i=0,j=0;
+	while(inputs[i])
+	{
+		if((inputs[i] >= '0')&&(inputs[i] <= '9'))
+		{
+			outputs[j++] = inputs[i];
+		}
+		i++;
+	}
+	outputs[j] = 0;
+	return 0;
+}
+
+int biggest(int *inputa, int size)
+{
+	int bige = inputa[0];
```

Comment:
"bige" does not look like a vaild variable name. Please make sure that your variable/function naming is clear and correct.